packages/block-editor/src/components/block-list/style.scss

```diff
@@ -326,6 +326,15 @@
 	z-index: z-index("{core/image aligned left or right} .wp-block");
 }
 
+.wp-site-blocks > [data-align="left"] {
```

Comment:
Would be cool to move at some point from these data-align attributes in the editor. Not sure how complex would that be

Reply:
What do you mean?

Reply:
The editor relies on these special data-align attributes (I guess it's for legacy themes) to style the alignments instead of using `alignleft` `alignright` classNames.

That said, @oandregal are these new styles necessary here? Since it's only about block themes, shouldn't the styles coming from the theme.json stylesheet already apply here?

Reply:
That was my thinking when I first prepared this. However, in the editor, the blocks get the `data-align` attributes instead of the align classes, hence the theme.json styles don't have any effect.

Is there any difference between classic and block themes when it comes to alignments? For reference, I've tested this with emptytheme (block theme) and twentytwenty (classic theme).

Reply:
> That was my thinking when I first prepared this. However, in the editor, the blocks get the data-align attributes instead of the align classes, hence the theme.json styles don't have any effect.

This could be left for later but maybe we should try adding the classes as well at first and see what it does.